app/views/shf_applications/_shf_application_fields.html.haml

```diff
@@ -46,11 +46,15 @@
 
       %h3.upload-button-pre= t('shf_applications.new.upload_files')
 
+
       %span{class: "btn btn-warning btn-file"}
         = t('shf_applications.new.upload_button')
         = f.file_field 'actual_files[]', multiple: true, name: 'uploaded_file[actual_files][]', class: 'upload-field', id: 'upload-button'
 
       - submit_label = @shf_application.new_record? ? t('submit') : t('save')
+      %p= t('shf_applications.new.upload_allowed_file_types')
+      %p= @allowed_file_types
```

Comment:
Please add an `accept` option to the `file_field` (line 51) so that the user can only select valid file types in the upload-file dialog box.

If you find a nice way to test this added item, it would be great.  But, that is not critical for this.

Reply:
BTW, adding the above item would also address https://www.pivotaltracker.com/story/show/153360586.

Reply:
@patmbolger the `accept` option was added to the `file_field` for the user to only see the allowed file types.